app/assets/javascripts/app/project/edit.js

```diff
@@ -29,3 +35,40 @@ App.addChild('ProjectEdit', _.extend({
   }
 }, Skull.Tabs));
 
+// FIXME: MORE DRY HERE !!
```

Comment:
@diogob the problem is the data-action generated when we use render :edit :/
or i just cant change the el to a wrapper class?